controllers/comments.controller.js

```diff
@@ -0,0 +1,16 @@
+const removeComment = require('../models/removeComment.model.js')
+
+exports.deleteComment = (req,res,next)=>{
+    const {comment_id} = req.params
+
+    removeComment(comment_id).then((deltedComment) => {
+        if (deltedComment !== undefined) {
+        res.status(204).send()
+        } else {
+        next({ status : 404, msg: "Not Found"})
```

Comment:
I am only going to comment on this once (even though I suspect this might be an issue for other tests of this kind too): You can save a lot of time by simply doing an error handler for all invalid endpoints - have a look at the last paragraph https://notes.northcoders.com/courses/js-back-end/error-handling